trappy/base.py

```diff
@@ -190,6 +200,16 @@ def create_dataframe(self):
                 expl_val = trace_parser_explode_array(val, trace_arr_lengths)
                 self.data_array[idx] = expl_val
 
+        bytes_free = _get_free_memory_bytes()
+        if bytes_free is not None:
+            # Pandas has been observed to segfault here when running out of
+            # memory. Let's make a very rough guess at whether that's likely and
+            # print a warning if we think it is.
+            dataframe_size_guess = 20 * 1024 * len(self.data_array)
```

Comment:
Where does the 20 * 1024 come from?  Every row in the data array uses 20Kb?  Can you add that to the comment?